main.js

```diff
@@ -24,4 +24,29 @@ for (let i = 0; i < mobileMenuLinks.length; i += 1) {
 
 closeTab.addEventListener('click', () => {
   closeMenu();
-});
\ No newline at end of file
+});
+
+const blurPop = document.querySelector('#blur');
+const popUp = document.querySelector('.popup');
+const seeProject = document.querySelector('.btn');
+
+function Toogle() {
+  blurPop.style.filter = 'blur(24px)';
+  popUp.style.display = 'block';
+}
+
+function closePopUp() {
+  popUp.style.display = 'none';
+  blurPop.style.filter = 'blur(0px)';
+}
+
+seeProject.addEventListener('click', () => {
+  Toogle();
+});
+
+const seeProjectLinks = document.querySelectorAll('.closePopUp');
+for (let i = 0; i < seeProjectLinks.length; i += 1) {
+  seeProjectLinks[i].addEventListener('click', () => {
+    closePopUp();
+  });
+}
```

Comment:
- [ ] The project requirements say that you should use a JavaScript array of objects to store the information for each project. Kindly use the example below 👇 
![Screenshot (91)](https://user-images.githubusercontent.com/67087664/147100407-0739d6f5-12f0-4830-9ffd-ca3eed516ecb.png)

- [ ] Also make sure the modal popup data is populated with the data from the projects array